CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 
 Unreleased
 ==========
+* Introduced Django 4.2 support.
+* Dropped Support for Django<3.1
+* Dropped the dependency of `aldryn-translation-tools`
```

Comment:
Dropped support for python<3.8

Reply:
@vipulnarang95 ,
add the missing changelog. 

Thanks for pointing this out.